src/main/java/org/ieeervce/gatekeeper/config/SecurityConfiguration.java

```diff
@@ -34,25 +39,31 @@ public PasswordEncoder passwordEncoder() {
     @Bean
     public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
         http.formLogin(Customizer.withDefaults()).httpBasic(Customizer.withDefaults()).authorizeHttpRequests(SecurityConfiguration::getCustomizedHttpAuthorization).csrf(AbstractHttpConfigurer::disable)
-                .cors(AbstractHttpConfigurer::disable);
+
+                .cors(customizer->customizer.configurationSource(corsConfigurationSource()));
```

Comment:
You're calling the method, while the bean is also present. This creates this object twice.
Either:
- Remove the @Bean on the function, OR
- Autowire the bean in using `@Qualifier`.